server/bin/www.js

```diff
@@ -125,4 +125,4 @@ async function onError(error) {
   }
 }
 
-run(port);
+module.exports.init = run(port);
```

Comment:
We should turn it around:

1. `server/init.js` should have the implementation logic of `www.js`; this is because that's the API that is exported for use downstream in the cli plugin devtool command
2. `bin/www` then does just `require('./bin/www.js').init`

Also we can just rename `server/init.js` to `server/index.js` and make that the main function. For a standard `require("@adobe/asset-compute-devtool")` include downstream.

Reply:
Also this already executes runs right away when this module is loaded (`run(port)` invocation). But clients should get control over that.

Also to suggest calling it `start()` as it starts a server (run suggests a transactional execution):
```
module.exports = {
    start: run
};
```

Now the `port` & `randomString` logic has to move into the `start/run()` function. I would suggest:

- `start(port)` takes `port` as an optional field (so that a client could control it if they want)
- inside `start()`, if `port` is not set, then it does `normalizePort(process.env.ASSET_COMPUTE_DEV_PORT || '9000'`
- `randomString` is simply moved inside `start/run()`

Reply:
@alexkli 
The developer does have control to set the port via ENV `ASSET_COMPUTE_DEV_PORT`. Is that sufficient or do you actually want it passed when the package.json script is executed?

Reply:
I think it's a bit of a philosophical discussion what component handles the env var:

- [cli](https://github.com/adobe/aio-cli-plugin-asset-compute): reads env var if set and passes it as argument to `start()` API
- devtool "library" here: supports argument in `start()` API but reads from env var if argument is not set

The latter is what I mentioned above, since it would mean the least changes. But either way are fine with me. At this point we don't really expect any other use of the devtool than in the [cli](https://github.com/adobe/aio-cli-plugin-asset-compute), so it doesn't matter much.